spec/models/apartment_spec.rb

```diff
@@ -1,5 +1,176 @@
 require 'rails_helper'
 
 RSpec.describe Apartment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:user) { User.create(
+    email: 'test@example.com',
+    password: 'password',
+    password_confirmation: 'password'
+    )
+  }
+
+  it 'should validate street' do 
+    apartment = user.apartments.create(
+    unit: "30", 
+    city: "New York",
+    state: "NY",
+    square_footage: 2000, 
+    price: 13000, 
+    bedrooms: 5, 
+    bathrooms: 3, 
+    pets: "Puppets only",
+    image: "https://upload.wikimedia.org/wikipedia/commons/0/00/Sesame_Street_buildings_%28193090534%29.jpg" 
+    )
+    expect(apartment.errors[:street]).to include("can't be blank")
+  end
+
+  it 'should validate unit' do 
+    apartment = Apartment.create(
```

Comment:
For this one and the rest of your tests, you are creating an Apartment without a user. While your test may pass with this, it is best practice to isolate the attribute you are testing for and ensure your test instance has all other attributes available. I would recommend updating these tests to be written like you have on line 12 (minus the test that is checking for user_id)